### DISCUSSION TITLE
How to modify <NuxtWelCome> component in Nuxt3?

### DISCUSSION THREAD

**Discussion by binbin0916** [nuxt/nuxt#30079] [Questions]
I want to modify the welcome page launched in nuxt3, but I couldn't find the corresponding documentation. Has anyone made any modifications to this?

**Comment by ndragun92** [comment]
![image](https://github.com/user-attachments/assets/7b5fb2e2-0cea-410a-aab8-697d4733dfb3)

There are props but you cannot modify how template looks like since its designed to be a placeholder and not something that you should use.

If you want to get HTML of it you can copy and paste it from
`node_modules/nuxt/dist/app/components/welcome.vue`

Everything inside that template is hardcoded even if you try to modify props itself

Hope it helps! @binbin0916

**Comment by JoJk0** [comment]
I've tried using `addComponent` from `@nuxt/kit` to override it:

```ts
import { addComponent, defineNuxtModule } from '@nuxt/kit'

export default defineNuxtModule(() => {
  addComponent({
    name: 'NuxtWelcome',
    export: 'NuxtWelcome',
    filePath: 'my-component-library/nuxt/utils',
    priority: 10,
  })
})
```

However I'm having a warning once I run Nuxt app with my custom module: 

```bash
 WARN  Overriding NuxtWelcome component. You can specify a priority option when calling addComponent to avoid this warning.
```

I'm not sure why I still have this warning as `priority` is being set on `addComponent`.
